charts/wire-server/requirements.yaml

```diff
@@ -115,3 +115,8 @@ dependencies:
   repository: "file://../sftd"
   tags:
     - sftd
+- name: integration
```

Comment:
probably a dumb question, but why is this not loaded in prod?

Reply:
Good question! It actually would! Good catch. Fixed in ddbc19771004a3770c7a4161f18a44abe6bec6b5

Reply:
https://github.com/wireapp/wire-server/commit/ddbc19771004a3770c7a4161f18a44abe6bec6b5 caused 

```
Error from server (NotFound): pods "test-la9l8u3xfu93-wire-server-integration-integration" not found
```
(see [logs](https://concourse.ops.zinfra.io/builds/20254955#L6423bec2:426))

We need to double-check, but I think 
```
  annotations:
    "helm.sh/hook": test
```
annotation in the `Pod` definition wouldn't deploy this pod in prod.